packages/fetchai/skills/carpark_client/skill.yaml

```diff
@@ -25,6 +25,9 @@ protocols:
 skills:
 - fetchai/generic_buyer:0.27.6
 behaviours:
+  check:
+    args: {}
+    class_name: ProposalCheckBehaviour
```

Comment:
It might make sense to expose the `ticker_interval` argument here to make that easier to configure.

Reply:
I think it's ok to have it exposed in the strategy as the `ticker_interval` itself is left default (1 sec) but the `proposal_check_interval` is more related to the overall behaviour of the agent.
I renamed it to `proposal_check_timeout` to make it more clear what I meant.